packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountCard/TransferFormAccountCard.scss

```diff
@@ -51,4 +51,14 @@
     &__modal-badge {
         margin-left: 1rem;
     }
+
+    .wallets-app-linked-with-wallet-icon {
+        .wallets-gradient {
+            border-radius: 0.15rem;
```

Comment:
hey @lubega-deriv 
where we are using 0.15rem border radius?

Reply:
hey @heorhi-deriv, I refered the figma files where the bigger currency icons has a border radius of 4px and smaller currency icons linked with wallet icon has a border radius of 1.5px

<img width="400" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/008a8a3f-7c21-423b-b536-237047b499d5">

<img width="400" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/dfb2b57a-d09e-483c-abf0-be00823a41ec">

Reply:
oh, ok @lubega-deriv 
thanks for clarification